src/test/java/org/apache/cassandra/sidecar/utils/XXHash32DigestVerifierTest.java

```diff
@@ -52,40 +51,37 @@ class XXHash32DigestVerifierTest
     @TempDir
     static Path tempDir;
     static Path randomFilePath;
+    static RestoreJobUtil restoreJobUtil;
 
     @BeforeAll
     static void setup() throws IOException
     {
         vertx = Vertx.vertx();
 
-        randomFilePath = TestFileUtils.prepareTestFile(tempDir, "random-file.txt", 1024);
-    }
+        restoreJobUtil = new RestoreJobUtil(new Lz4XXHash32Provider());
 
-    @Test
-    void failsWhenDigestIsNull()
```

Comment:
This test is removed, as the test case is imaginary. In prod code, the verifiers are created by the static create method, which guarantees the digest is non-null.